### DISCUSSION TITLE
Tunes

### DISCUSSION THREAD

**Discussion by kishonadiaz** [codex-team/editor.js#2253] [Q&A] [answered]
I am reading the doc on Tunes but it does not specify how to implement it into editorjs can anyone explain

**Comment by neSpecc** [comment, marked as answer]
Take a look at this example: https://github.com/editor-js/text-variant-tune

**Reply by kishonadiaz** [reply]
Thank you for your response I tried that way of doing it didn't work I found another way thank you so much
